pyproject.toml

```diff
@@ -52,20 +56,18 @@ pylint-django = "*"
 yamllint = "*"
 Markdown = "*"
 # Rendering docs to HTML
-mkdocs = "1.3.1"
+mkdocs = "1.5.2"
 # Material for MkDocs theme
-mkdocs-material = "8.4.2"
+mkdocs-material = "9.2.4"
 # Render custom markdown for version added/changed/remove notes
 mkdocs-version-annotations = "1.0.0"
 # Automatic documentation from sources, for MkDocs
 mkdocstrings = "0.22.0"
-mkdocstrings-python = "1.1.2"
-# griffe 0.31 and later drop support for python 3.7
-griffe = "0.30.1"
+mkdocstrings-python = "1.5.2"
 
 [tool.black]
 line-length = 120
-target-version = ['py37']
+target-version = ['py311']
```

Comment:
feel like we'd want black target-version set to lowest supported py version?

Reply:
why? Perhaps that is correct, but not sure why tbh.

Reply:
was thinking it was what I read the most in https://github.com/psf/black/issues/751
I suppose it doesn't matter as much as I was thinking.